HyperVExtension/src/DevSetupAgent/DevSetupAgent.csproj

```diff
@@ -24,13 +24,19 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Windows.CsWinRT" Version="2.0.4" />
+    <PackageReference Include="Serilog" Version="3.1.1" />
+    <PackageReference Include="Serilog.Extensions.Logging" Version="8.0.0" />
+    <PackageReference Include="Serilog.Settings.Configuration" Version="8.0.0" />
+    <PackageReference Include="Serilog.Sinks.Console" Version="5.0.1" />
+    <PackageReference Include="Serilog.Sinks.Debug" Version="2.0.0" />
+    <PackageReference Include="Serilog.Sinks.File" Version="5.0.0" />
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\common\DevHome.Common.csproj" />

```

Comment:
I only add it because DevHome Common is where I put the log location definitions. I could duplicate that code into the other embedded extensions, but I thought having it defined once and referenced by the extensions sharing the common project would be best.

I haven't thought about it beyond that. I could duplicate the definitions if we want to not bring Common in, but my preference is to not duplicate logging definitions anymore.  Does adding Common into this project cause a problem?

Reply:
@bbonaby @EricJohnson327 Is there any reason we want to keep DevHome.Common out of extensions?